scripts/genera_pdf.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#author: Simone Massaro (mone27)
+#date: 15/05/18
+#this scripts takes html file converts to single pdf pages and then merges the pages
+#and try to install all dependencies (tested on ubuntu)
+#now paths are hardcoded for firefox-vademecum repository
+if ! [ -x "$(command -v html-pdf)" ]; then
+  echo 'html-pdf not installed installing..'
+  if ! [ -x "$(command -v npm)" ]; then
+    echo "npm not installed. Installing..."
+    curl https://www.npmjs.org/install.sh | sh
+  fi
+  npm install html-pdf
```

Comment:
così credo che lo installa nella cartella in cui si sta e non globale, credo che serva il parametro `-g`

Reply:
non lo so non ho avuto modo di testare